js-exercises/permutate-without-repetitions/permutateWithoutRepetitions.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @param {*[]} permutationOptions
+ * @return {*[]}
+ */
+function permutateWithoutRepetitions(permutationOptions) {
+	
+}
+
+export { permutateWithoutRepetitions };
```

Comment:
Looks like you didn't push the changes for this problem. Please push them. Thanks.